lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -96,13 +96,14 @@ sub validate_birth_date {
     return $field->add_error("invalid date") unless Date::Calc::check_date($year, $month, $day);
 }
 
-sub _requires_email {
+sub _requires_not_limited {
     my ($self, $field) = @_;
-    return $field->add_error(l('Biography and website fields may not be edited until your email address is confirmed.')) unless $self->ctx->user->has_confirmed_email_address;
+    return $field->add_error(l('Spammers are not welcome, go away.')) unless $self->ctx->user->is_limited;
+    ;
```

Comment:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/515/commits/76c82854c6558e8dfac0100266fa31926db59c59